src/components/Slider/RangeSlider/RangeSlider.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import {
+  Root as SliderRoot,
+  Track as SliderTrack,
+  Range as SliderRange,
+  Thumb as SliderThumb
+} from '@radix-ui/react-slider';
+
+import styles from '../Slider.module.scss';
+import classNames from 'classnames/bind';
+const cx = classNames.bind(styles);
+
+export interface RangeSliderProps {
+  values?: number[];
+  min: number;
+  max: number;
+  step: number;
+  minStep?: number;
+  isSmall?: boolean;
+  isLight?: boolean;
+}
+
+export const RangeSlider: React.FC<RangeSliderProps> = ({ step, min, max, values, isSmall, minStep, isLight }) => {
+  const [currentValues, setCurrentValues] = useState(values ?? [min, max]);
+  return (
+    <div className={cx(styles.RootContainer, { LightRootContainer: isLight })}>
+      {currentValues[0]} - {currentValues[1]}
+      <form>
+        <SliderRoot
+          className={cx(styles.SliderRoot, { SmallRoot: isSmall, LightRoot: isLight })}
+          defaultValue={[min, max]}
+          min={min}
+          max={max}
+          step={step}
+          minStepsBetweenThumbs={minStep}
+          aria-label="Range slider"
```

Comment:
`aria-label` is used to describe what an element is for - `aria-label` should be provided by the parent to give context, e.g. `aria-label="Temperature slider"`, `aria-label="Age slider"`. If there were 5 range sliders on a page, screen readers might have a tough time figuring out which is which if they're all called `"Range slider"`.